src/main/java/org/folio/rest/impl/ExchangeApi.java

```diff
@@ -26,4 +26,14 @@ public void getFinanceExchangeRate(String from, String to, Map<String, String> o
       .onSuccess(body -> asyncResultHandler.handle(succeededFuture(Response.ok(body, APPLICATION_JSON).build())))
       .onFailure(t -> handleErrorResponse(asyncResultHandler, helper, t));
   }
+
+  @Override
+  public void getFinanceCalculateExchange(String from, String to, Number amount,
+                                          Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler,
+                                          Context vertxContext) {
+    ExchangeHelper helper = new ExchangeHelper(vertxContext);
+    helper.calculateExchange(from, to, amount)
```

Comment:
Basically seems we need to user to wrap in execute blockin construction(not derecated one if possible) , under the hood the call to Java money API is made and it is syncronus operation, Vertx has not an asynchronus client for it, I think better to wrap